### DISCUSSION TITLE
error during installation

### DISCUSSION THREAD

**Discussion by RuanMD** [AppFlowy-IO/AppFlowy#1749] [General help wanted]
I had an installation error, what can I do to solve it?
![app](https://user-images.githubusercontent.com/44640884/215261667-0ead794f-4292-4e84-92e6-9ef722d677e6.png)

**Comment by LucasXu0** [comment]
Hi, @RuanMD. AppFlowy project uses Flutter 3.3.x now. So downgrading the flutter version will solve it.
